reactapp/utils.js

```diff
@@ -0,0 +1,60 @@
+const child_process = require("node:child_process");
+const http = require("node:http");
+
+const finalhandler = require("finalhandler");
+const getPort = require("get-port");
+const serveStatic = require("serve-static");
+const treeKill = require("tree-kill");
+
+exports.getPort = getPort;
+
+exports.exec = (command, env, cwd) => {
+  const proc = child_process.exec(command, {
+    env: { ...process.env, ...env },
+    cwd,
+  });
+
+  proc.on("error", (error) => {
+    console.log(error);
+  });
+
+  // proc.stdout.setEncoding("utf8");
+  // proc.stdout.on("data", (data) => {
+  //   console.log(data);
+  // });
+
+  // proc.stderr.setEncoding("utf8");
+  // proc.stderr.on("data", (data) => {
+  //   console.log(data);
+  // });
```

Comment:
The information from the `dev` or `build` command could be useful for debugging. I was trying out the library and got stuck momentarily because I didn't realize I forgot to run `npm install` in the react app folder. 😅 

Maybe it's worth at least logging the stderr by default:
```suggestion
  // proc.stdout.setEncoding("utf8");
  // proc.stdout.on("data", (data) => {
  //   console.log(data);
  // });

  proc.stderr.setEncoding("utf8");
  proc.stderr.on("data", (data) => {
    console.log(`[${command}] ${data}`);
  });
```